src/sidebar/barrel/tree/BarrelTreeDataProvider.ts

```diff
@@ -9,6 +9,9 @@ import SidebarRefresher from "../../refresh/util/SidebarRefresher";
 import AddBarrelTreeItem from "./AddBarrelTreeItem";
 import BarrelType from "../../../common/domain/barrel/BarrelType";
 import localization from "../../../localization";
+import { isVscodeVersionSufficient } from "../../../common/vscode/ide/vscodeUtil";
+
+const VIEWS_WELCOME_MIN_VERSION = "1.44.0";
```

Comment:
It would be great to collect min versions for features in one file.

Reply:
👍  let me open a separate PR for that

Reply:
https://github.com/zeplin/vscode-extension/pull/72